### ISSUE TITLE
How to get the relevance training samples for Classification of a picture as a new training sample?

### ISSUE THREAD

**Issue by MatheCore** [pytorch/captum#671]
Hello,

I am actually new in this feld . I used IntegratedGradients to interpretate an image Classifier. Now i would like to get the relevance pixels (training samples) as a new training set. With ignoring the rest so i get a much smaller training set. How could i do that?

I appreciate any help :)

Thanks

**Comment by bilalsal** [comment]
Hi @MatheCore,

if I understand correctly, you want to use the attribution map A computed by IntegratedGradients for an image X as a pixel mask?
Technically you can do that, e.g. by thresholding A (say into A') and then multiplying A' * X pixel-wise.
I am unsure the result would be informative, as it might fundamentally change the image features.
